approved/http.yaml

```diff
@@ -0,0 +1,35 @@
+- action:
+    uses: harness/http-arm
+    with:
+      url: https://webhook.site/2561218a-36d4-4b9c-83b0-bb89fb64ebc8
```

Comment:
url -> wil lbe host, and does e.g. value should go here

Reply:
Yes the URL value should be there.